src/components/RegisterLoginModal/RegisterLoginModal.jsx

```diff
@@ -1,15 +1,34 @@
-import { useState } from 'react';
-
+/* eslint-disable react/no-children-prop */
+/* eslint-disable no-shadow */
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-absolute-path */
+/* eslint-disable import/no-unresolved */
```

Comment:
I think something could be done with the 3 rules above so please consider also removing this.